src/components/Book.js

```diff
@@ -14,17 +14,56 @@ function Book({ title, author, id }) {
   };
 
   return (
-    <li className={styles.singleBook}>
-      <p>{title}</p>
-      <p>{author}</p>
-      <button
-        data-id={id}
-        type="button"
-        onClick={removeHandler}
-      >
-        remove
-      </button>
-    </li>
+    <>
+      {/* <li className={styles.singleBook}>
+        <p>{title}</p>
+        <p>{author}</p>
+        <button
+          data-id={id}
+          type="button"
+          onClick={removeHandler}
+        >
+          remove
+        </button>
+      </li> */}
```

Comment:
- [ ] Great work with styling the app, however I cannot yet approve the project as you have commented code in your codebase following the JS best practices it is recommended to not commit old pieces of code as inline comments. They will make the project look messy. please fix this